### DISCUSSION TITLE
How to change grub theme?

### DISCUSSION THREAD

**Discussion by Disil** [SpiralLinux/SpiralLinux-project#289] [Q&A]
As the title says, how do I change the grub theme so I can use other themes instead of the green background? I already installed `grub-theme-breeze` but the grub menu still shows the stock SpiralLinux one.

**Comment by geckolinux** [comment]
Hi there, please try the GRUB Customizer utility that comes pre-installed.

**Reply by Disil** [reply]
can I change it without grub customizer?

**Reply by robertocannito** [reply]
Remove splash.png from /boot/grub
sudo update-grub
reboot and you will find emarald theme from debian
at that point install theme that you want folloing istruction included with theme

Me too, If I can avoid to use Grub Customizer, sametime it may be very dangerous
